bin/nib

```diff
@@ -8,6 +8,35 @@ import sys
 import nib
 from nib.config import default_config
 
+if sys.version_info[0] == 2:
+    input_func = raw_input
+else:
+    input_func = input
+
+
+def getYN(question, dflt=True):
```

Comment:
Sorry, it's a reserved world in some other languages I work in, habit.  Kind of like using `klass` instead of `class.`  Done.